src/cbc_sdk/base.py

```diff
@@ -58,11 +58,11 @@ def __new__(mcs, name, bases, clsdict):
             model_data = yaml.safe_load(
                 open(os.path.join(mcs.model_base_directory, swagger_meta_file), 'rb').read())
 
-        clsdict["__doc__"] = "Represents a %s object in the Carbon Black server.\n\n" % (name,)
-        for field_name, field_info in iter(model_data.get("properties", {}).items()):
-            docstring = field_info.get("description", None)
-            if docstring:
-                clsdict["__doc__"] += ":ivar %s: %s\n" % (field_name, docstring)
+        # clsdict["__doc__"] = "Represents a %s object in the Carbon Black server.\n\n" % (name,)
```

Comment:
Would really like confirmation that removing these lines hasn't broken anything.  It hasn't from what I could see, but could be unexpected side affects.  Commented pending agreement it can really be removed.

Reply:
I'm not sure what's going on here.  It looks like the metaclass is trying to b0rk around with the class docstring based on information in the Swagger metafile. `:ivar` is a Sphinx-format documentation directive. If this is to be reused, we should determine how instance-variable documentation works in Google-format doc strings and do that.

Reply:
I've replaced that chunk of code with something that expresses the intent of what that original code was trying to do, but does it following the Google format, and also preserves any existing docstring that was already in the code.  Try generating the docs yourself and let me know what you think.